src/esm_tests/info.py

```diff
@@ -0,0 +1,165 @@
+import argparse
+import os
+
+class Info(dict):
+    """
+    Dictionary subclass that contains all the information about the tests. The
+    information is stored under the following `keys`:
+
+    - ``actually_compile``: ``True`` if the test should compile or ``False`` if it
+      should run a check instead.
+
+    - ``actually_run``: ``True`` if the test should run or ``False`` if it should
+      run a check instead.
+
+    - ``bulletpoints``: when printing results or the state should it use bulletpoints
+      for the items or no.
+
+    - ``system_exit_on_errors``: triggers a system exit on errors or file differences
+      so that GitHub actions can catch that as a failing test (defined by the ``-e``
+      argument).
+
+    - ``hold``: should it wait for user input after each test or not.
+
+    - ``ignore``: lines to be ignored during comparison, loaded from
+      ``<PATH>/esm_tools/src/esm_tests/resources/ignore_compare.yaml``.
+
+    - ``ignore_user_info``: user-specific Info to remove from the files ``last_tested``
+      files.
+
+    - ``in_github``: indicates whether it is running in GitHub or not (defined by the
+      ``-g`` argument).
+
+    - ``last_tested_dir``: directory of the configuration files for the last tested
+      experiments (i.e. ``<PATH>/esm_tools/src/esm_tests/resources/last_tested/``).
+
+    - ``rm_user_info``: User-specific Info to remove from the files ``last_tested``
+      files.
+
+    - ``repo_update``: does it need to update the ``resources`` or not
+      (specified by the user).
+
+    - ``scripts``: dictionary including all information of every runscript to be run
+      in the current computer.
+
+    - ``script_dir``: directory of the ``esm_tests`` code, useful to call the
+      ``resources`` and the ``resources/runscripts``.
+
+    - ``this_computer``: name of the current computer
+
+    - ``user``: ``esm_tests`` configurations specific to the user, as read from
+      ``<PATH>/esm_tools/src/esm_tests/user_config.yaml``.
+
+    """
+
+
+    def __init__(self):
+        self["script_dir"] = os.path.join(os.path.dirname(os.path.realpath(__file__)), ".")
+
+
+    def argparse(self):
+        """
+        Parses the arguments from the ``esm_tests`` call.
+
+        Returns
+        -------
```

Comment:
```suggestion
        -------
         Returns a tuple with the following entries:
```

Check how that renders in the documentation.

Reply:
The dosctring building in the docs is broken since long time, so I cannot check it. I tried a couple of times to try to fix it and failed.